.travis.yml

```diff
@@ -2,11 +2,9 @@ sudo: false
 language: node_js
 node_js:
   - stable
-  - '5'
+  - '6'
```

Comment:
maybe add other recent versions of node as well?

Reply:
I'll switch it away from "stable" and use all evens like I did in the other repos.